public/jsx/components/input-error.jsx

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var React = require('react');
+
+var InputError = React.createClass({
+  displayName: 'InputError',
+
+  propTypes: {
+    optModifier: React.PropTypes.oneOf(['center', 'right', 'left']),
+    text: React.PropTypes.string.isRequired,
+  },
+
+  render: function() {
+    var { text, ...toolTipAttrs } = this.props;
+
+    var errorClassList = ['tooltip'];
+    errorClassList.push(this.props.optModifier || 'left');
+    var errorClass = errorClassList.join(' ');
```

Comment:
Could do - I used it earlier when switching to react. For whatever reason it's not built-in now, it's another lib. Given this is small I'll just do it manually. If I get to the point that this is becoming a repeating pattern it'll be worth adding the classSet dep